static/app/views/dashboards/widgetBuilder/components/newWidgetBuilderSlideout.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import SlideOverPanel from 'sentry/components/slideOverPanel';
+import {IconClose} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+type WidgetBuilderSlideoutProps = {
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+export default WidgetBuilderSlideout;
+
+function WidgetBuilderSlideout({isOpen, onClose}: WidgetBuilderSlideoutProps) {
```

Comment:
Can we rename this file to `widgetBuilderSlideout.tsx`? The `new` isn't really necessary